src/solve.jl

```diff
@@ -102,72 +116,23 @@ Interface to reachability algorithms for a hybrid system PWA dynamics.
 - `system`  -- hybrid system
 - `options` -- options for solving the problem
 """
-function solve(system::InitialValueProblem{<:HybridSystem, <:LazySet{N}},
-               options::Options)::AbstractSolution where N<:Real
-    sys_new = init_states_sys_from_init_set_sys(system)
-    opC, opD = default_operator(sys_new)
-    return solve!(sys_new, copy(options), opC, opD)
-end
-
-function solve(system::InitialValueProblem{<:HybridSystem, <:LazySet{N}},
+function solve(system::InitialValueProblem{<:HybridSystem, S},
                options::Options,
-               opC::ContinuousPost,
-               opD::DiscretePost
-              )::AbstractSolution where N<:Real
-    sys_new = init_states_sys_from_init_set_sys(system)
+               opC::ContinuousPost=BFFPSV18(),
+               opD::DiscretePost=LazyDiscretePost())::AbstractSolution where {N, S}
+    sys_new = distribute_initial_set(system)
```

Comment:
If you want to call this function in all cases, you need to define a method that is just the identity in case the initial states are already given as the list of tuples.

If there is no test case capturing this, please add one.

We may want to allow more types of inputs for the initial states (e.g., a dictionary) and then internally convert to some normal form, but this we can do later.

Reply:
i've updated `solve` to do the distribution of the initial state over the locations if needed, otherwise go ahead.